subjects/gaming/stealth-boom/audit/README.md

```diff
@@ -24,46 +14,70 @@
 
 ###### Are the mouse settings (mouse sensitivity, invert mouse vertical axis) functioning according to their descriptions?
 
-###### Do the guards and drones wander around the map?
+##### Map / Level
 
-###### When a player enters the field of view of a guard, does he switches to chasing mode, running and shooting towards the player while also taking cover?
+###### Does the map contain props/walls for the player to hide from enemies?
 
-###### Does the drone light switches between each state? Blue for patrolling and red for chase mode (when a player crosses its sight)?
+###### Does the map contain pickable ammunition and health placed around?
 
-###### Whenever a drone turns to chasing mode, do all the guards in the area get alerted and switch to chasing mode as well?
+##### Player Mechanics
 
-###### Does the drone come back to patrol mode when the player is out of the drone sight?
+###### Does the player have all the minimal animation required (walking, running, melee attacking, shooting, ducking)?
 
-###### Is the sight radius of the drones smaller than the guards?
+###### Is there a sound for the guns shooting?
 
-###### Are cameras attached to walls?
+###### Is there a widget displaying the amount of bullets the player has?
 
-###### Do cameras have similar light sign as the drones (red for alert mode and blue for patrol)?
+###### Is the player unable to shoot when he has no bullets?
 
-###### As the drones, do the cameras alert guards on the area, switching them to chasing mode?
+###### Are there bullets impacts present when shooting at a wall?
 
-###### Do Guards, Drones and Cameras play an alert sound when a player gets detected?
+###### Does the player health decreases when he gets damaged?
 
-###### Do some cameras lock access to certain areas of the map, when they detect a player?
+###### When the player is hit, are there any blood particles?
 
-###### Can the camera close some part of the map (thru closed doors, open traps and tries to kill the player etc…) to the player when the player is being detected?
+###### Does the player have a health bar?
 
-###### Are Behavior Trees used to implement the AI of the Guards and Drones?
+###### When the player loses all his health (dies), does he get to choose whether to quit the game, go back to the main menu, or to start over?
 
 ###### Does the player have a goal?
 
 ###### When the goal of the player is successfully completed, does a pop up appear saying that the mission is completed?
 
```

Comment:
Separate the `Gun mechanics:`, organize questions and add missing question here is a suggestion:

```
##### Player Mechanics

###### Does the player have all the minimal animation required (walking, running, melee attacking, shooting, ducking)?

###### Does the player have a health bar?

###### Does the player health decreases when he gets damaged?

###### When the player is hit, are there any blood particles?

###### When the player loses all his health (dies), does he get to choose whether to `quit the game`, go back to the `main menu`, or to `start over`?

###### Does the player have a defined goal or mission?

###### Whatever the goal or mission, can you confirm that the player had enemies on his way to divert him away from his objective?

###### When the player successfully completes the goal or mission, does a pop-up appear indicating `mission completion`?

#### Gun Mechanics

###### Is there a sound for the guns shooting?

###### Is there a widget displaying the amount of bullets the player has?

###### Is the player unable to shoot when he has no bullets?

###### Are there bullets impacts present when shooting at a wall?

```

Reply:
seperated.